### PR TITLE
WIP; check if formplayer can talk to couch

### PR DESCRIPTION
**Pull request by sravfeyn**

### CODE REVIEW

File: src/main/java/application/UtilController.java [hunk 1]
```diff
@@ -64,6 +65,12 @@ public ServerUpBean serverUp() throws Exception {
         return new ServerUpBean();
     }
 
+    @ApiOperation(value = "Check whether formplayer can connect to couch")
+    @RequestMapping(value = Constants.URL_COUCH_CONNECTION, method = RequestMethod.GET)
+    public String serverUp() throws Exception {
+        return CouchUserRepo.couchIsUp();
```
Comment (sravfeyn) [inline src/main/java/application/UtilController.java]: How can I wrap this under some kind of HTTP reponse object?

File: src/main/java/repo/impl/CouchUserRepo.java [hunk 2]
```diff
@@ -33,4 +33,8 @@ public CouchUser getUserByUsername(String username) {
         }
         return users.get(0);
     }
+
+    public String couchIsUp() {
+        // Test whether requests to couch are going
```
Comment (sravfeyn) [inline src/main/java/repo/impl/CouchUserRepo.java]: I am trying to query `couchDbClient` for `couch_host/couch_db_name` and test whether the response JSON contains the key-val `{'db_name': <couch_db_name>}`.

Can you give an example of how to query `couchDbClient` by providing a URL? And an example of looking up properties of JSON objects in Java.

**Comment by benrudolph** [comment]
agree with @snopoke and then we just need to update the health check in django to hit and parse that endpoint

**Comment by wpride** [comment]
Formplayer no longer talks to couch - 🎉

**Pull request closed without merging**